tests/functional_small/solvers_nonlinear/newton_raphson/eigen_w_basic_backtrack.cc

```diff
@@ -0,0 +1,70 @@
+//
+#include "pressio/solvers.hpp"
+struct ValidSystem 
```

Comment:
can you please write here where you found this, and a bit more details what you are solving? 
so that 1 year from now we can pick this up and undestand what is going on

Reply:
Added details on the problem, it's from here: http://www.math.iit.edu/~fass/477577_Chapter_17.pdf